### ISSUE TITLE
Issue with creating or modifying excel tables

### ISSUE THREAD

**Issue by mariruilo** [TillMacher/apscale_gui#7]
Hi, 

While working with Apscale I have issues with generating or modifying the Excel files. For the settings file, I can overcome the problem by modifying it manually instead of doing it using the program. I can run all the pipeline, however when I want to create the taxonomy table it apparently creates it, but then I have an error message and I can't open it. 

Thanks for your help.

**Comment by TillMacher** [comment]
Hi,
which OS and software for opening the Excel files are you using? This can sometimes happen with Open or libre office.
cheers
Till

**Comment by mariruilo** [comment]
Hi, 

I'm using a MacOS  Ventura 13.1 with M1 chip. I use microsoft Excel for opening the files. 

Thanks for the help. 

Mari

**Comment by TillMacher** [comment]
That's weird. I use the same system and this problem never occurred to me. What happens when you look at the file in preview mode (using the space key)? Can you see the content of the taxonomy table?
Maybe you could try and update your system to Ventura 13.5 (this should be the latest version) and try updating Excel?

**Comment by mariruilo** [comment]
This is what I see 

<img width="2240" alt="Captura de pantalla 2023-09-29 a las 9 16 21" src="https://github.com/TillMacher/apscale_gui/assets/3029208/bccb29ff-ea52-4afb-b98f-63b6036e8709">

**Comment by TillMacher** [comment]
The file is empty. So Apscale seems to not write them properly. But there is no Error message printed by Apscale when you run the pipeline?
Can you please check what the log file. Maybe all your reads are getting removed at some point in the analysis?

**Comment by MerjanK** [comment]
Hi, @mariruilo 
Maybe not my place to suggest this but I've had a similar problem with the excel files as well. Can you also check which pandas version you're using? I have had pandas 2.1.0 so the writer.save() is not used in the newer versions. But when i downgraded it to pandas 1.5.3, I was able to save the settings file. But I am not so sure about the taxonomy table error. But this might be the reason why your excel files are getting corrupted and cannot be opened.

**Comment by TillMacher** [comment]
Yeah I also noticed that in the latest pandas version writer.save() has been removed. I'll fix that in the next update! Until then you'll have to downgrade, as suggested above!
